lib/ui_constants.dart

```diff
@@ -0,0 +1,18 @@
+abstract class UiConstants {
+  static const double firstPositionedBottom = 0;
+  static const double firstPositionedRight = 20;
+  static const double secondPositionedTop = 15;
+  static const double secondPositionedRight = 15;
+  static const double thirdPositionedWidth = 350;
+  static const double thirdPositionedTop = 170;
+  static const double thirdPositionedRight = 10;
+  static const double sizedBoxHeight = 20;
+  static const double iconSize = 55;
+  static const double fontSize25 = 25;
+  static const double fontSize20 = 20;
+  static const double padding16 = 16.0;
+  static const double padding8 = 8.0;
+  static const double padding150 = 150.0;
```

Comment:
Avoid to use number on const definitions

Reply:
done